### ISSUE TITLE
Its a real shame

### ISSUE THREAD

**Issue by litzinger** [ilikenwf/nestedSortable#117]
This appears to be abandoned code with no updates in 2 years. To top it off the docs refer to two config options "tolerenceElement" and "handle", and neither of those options are even in the source code, so apparently the only way for this library to work is if you're using the markup from the example. Forget about customizing the markup to something else.

**Comment by Bob-Lob-Law** [comment]
I agree it's a shame this has not been maintained.

In reference to your "tolerenceElement" and "handle" comment, maybe I can help. These two properties are set within the page itself. Have a look and the source in the example.html (lines 185 && 193).

**Comment by litzinger** [comment]
Yeah, those options are in the config when instantiating the plugin, but look at the source code, they don't exist, so those config items aren't doing anything.

**Comment by Johnniey** [comment]
They are not useless, they are used by jQuery UI, not this plugin.
See: [jQuery UI Sortable](https://github.com/jquery/jquery-ui/blob/74f8a0ac952f6f45f773312292baef1c26d81300/ui/widgets/sortable.js#L117)

**Comment by ilikenwf** [comment]
If I were to pass the project along would one of you want to maintain and update it, then?

**Comment by litzinger** [comment]
Too many other things to maintain, sorry.

**Comment by aleray** [comment]
@litzinger @Bob-Lob-Law:

<https://mikemcquaid.com/2018/03/19/open-source-maintainers-owe-you-nothing/>

**Comment by litzinger** [comment]
Sweet passive aggressive comment.

**Comment by tomkiernan120** [comment]
@ilikenwf I would be interested in maintaining, this is something I am really look for in a current project.

**Comment by ilikenwf** [comment]
Would you want me to just note a transfer/fork in the readme, or do you
just want admin rights on this repo? I can do either one.

**Comment by tomkiernan120** [comment]
@ilikenwf could you quote to a transfer/fork, personally I think it may be the best way. But open to help/advice if you have any

**Comment by tomkiernan120** [comment]
@ilikenwf I have created a fork now at [https://github.com/tomkiernan120/nestedSortable](https://github.com/tomkiernan120/nestedSortable)
